compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -359,7 +359,11 @@ fn run_compiler(
         // printing some information without compiling, or exiting immediately
         // after parsing, etc.
         let early_exit = || {
-            if let Some(guar) = sess.dcx().has_errors() { Err(guar) } else { Ok(()) }
+            if let Some(guar) = sess.dcx().has_errors() {
+                Err(guar)
+            } else {
+                Ok(())
+            }
```

Comment:
This and a bunch of other formatting changes get reverted by a later commit. It would be nicer for reviewing and `git blame` if you could amend this commit to remove the formatting changes.

Reply:
For some reason my format on save is different than the one from `x fmt`, do you know why that might be?